starlite/kwargs.py

```diff
@@ -499,26 +508,94 @@ def to_kwargs(self, connection: Union["WebSocket", "Request"]) -> Dict[str, Any]
 
         return output
 
-    async def resolve_dependency(
-        self, dependency: "Dependency", connection: Union["WebSocket", "Request"], **kwargs: Any
-    ) -> Any:
-        """Given an instance of [Dependency][starlite.kwargs.Dependency], recursively resolves its dependency graph.
+    async def resolve_dependencies(self, connection: Union["WebSocket", "Request"], kwargs: Dict[str, Any]) -> None:
+        """Resolve all dependencies into the kwargs, recursively.
 
         Args:
-            dependency: An instance of [Dependency][starlite.kwargs.Dependency]
             connection: An instance of [Request][starlite.connection.Request] or [WebSocket][starlite.connection.WebSocket].
-            **kwargs: Any kwargs to pass recursively.
+            kwargs: Kwargs to pass to dependencies
```

Comment:
```suggestion
            **kwargs: Kwargs to pass to dependencies
```

This is the correct format. You can test for doc issues by running `docker compose up --build`, which will bring the docs up locally.

Reply:
I don't think the ** is correct? The kwargs are passed in as a dictionary, so that they can be mutated.